devicemodel/samples/apl-mrb/launch_uos.sh

```diff
@@ -216,6 +216,14 @@ echo "8086 5abc" > /sys/bus/pci/drivers/pci-stub/new_id
 echo "0000:00:18.0" > /sys/bus/pci/devices/0000:00:18.0/driver/unbind
 echo "0000:00:18.0" > /sys/bus/pci/drivers/pci-stub/bind
 
+# Check if the NPK device/driver is present
+ls -d /sys/bus/pci/drivers/intel_th_pci/0000* 2>/dev/null 1>/dev/null
+if [ $? == 0 ];then
+  npk_virt="-s 0:2,npk,8/24"
```

Comment:
Note that for android, gvt-g has already taken slot 2, can't use 0:2 for npk virt device here.

Reply:
Based on the code of pci_parse_slot(), "0:2" here indicates "0:0:2", so it doesn't conflict with "0:2:0" for gvt-g. But should I use "0:0:2" here to avoid confusion?

Reply:
Then either way is OK. It's up to you.